### PR TITLE
[ADD] feat(submission): Allow help link for any form fields.

### PR DESCRIPTION
**Pull request by zannkukai**
If the backend forms specify an help url link, a new icon is now visible before the field label. Cliking on the icon redirect the user to the corresponding link into new brower tab.

Authored-by: Renaud Michotte <renaud.michotte@uclouvain.be>

### CODE REVIEW

File: src/app/shared/form/builder/ds-dynamic-form-ui/ds-dynamic-form-control-container.component.ts [hunk 1]
```diff
@@ -395,6 +395,7 @@ export class DsDynamicFormControlContainerComponent extends DynamicFormControlCo
   }
 
   get isCheckbox(): boolean {
+    console.log(this.model.type, this.model.label);
```
Comment (Pourbaix) [inline src/app/shared/form/builder/ds-dynamic-form-ui/ds-dynamic-form-control-container.component.ts]: Remove console.log

File: src/app/shared/form/builder/ds-dynamic-form-ui/models/ds-dynamic-input.model.ts [hunk 2]
```diff
@@ -62,13 +63,15 @@ export class DsDynamicInputModel extends DynamicInputModel {
   @serializable() toggleSecurityVisibility = true;
   @serializable() isModelOfNotRepeatableGroup = false;
   @serializable() hideHint: boolean = false;
+  @serializable() help: string | null;
```
Comment (Pourbaix) [inline src/app/shared/form/builder/ds-dynamic-form-ui/models/ds-dynamic-input.model.ts]: Why not use `@serializable() help?: string;` instead ?

File: src/app/shared/form/builder/parsers/field-parser.ts [hunk 3]
```diff
@@ -310,7 +310,7 @@ export abstract class FieldParser {
     }
   }
 
-  protected initModel(id?: string, label = true, labelEmpty = false, setErrors = true, hint = true) {
+  protected initModel(id?: string, label = true, labelEmpty = false, setErrors = true, hint = true, help = true) {
```
Comment (Pourbaix) [inline src/app/shared/form/builder/parsers/field-parser.ts:313]: maybe use a more explicit variable name like 'showHelp' instead of just 'help'

**Review by Pourbaix — CHANGES_REQUESTED** [review]
Some little changes here and there (I promise I'm not trying to be picky :smiling_imp:).

File: src/app/shared/form/builder/parsers/field-parser.ts (hunk 3 above)
Comment (zannkukai) [reply src/app/shared/form/builder/parsers/field-parser.ts:313]: `help` is the name the field (like 'hint', I keep the same logic)

**Review by Pourbaix — APPROVED** [review]

**Pull request merged**